src/device.rs

```diff
@@ -326,7 +341,7 @@ pub extern "C" fn wgpu_device_create_render_pipeline(
 }
 
 #[no_mangle]
-pub extern "C" fn wgpu_device_destroy_render_pipeline(render_pipeline_id: id::RenderPipelineId) {
+pub extern "C" fn wgpu_destroy_render_pipeline(render_pipeline_id: id::RenderPipelineId) {
```

Comment:
This is now inconsistent with the other destroy methods (others have `_destroy` at the end).

Reply:
Oops, good catch. Let me fix that right now